pyfr/solvers/baseadvec/kernels/transform.mako

```diff
@@ -0,0 +1,81 @@
+<%namespace module='pyfr.backends.base.makoutil' name='pyfr'/>
+
+// Transforms to m = [1, 0, 0]^T
```

Comment:
Mako comment.  Also not sure what m is.

Reply:
Reference to the paper that it came from I think